xwiki-platform-core/xwiki-platform-index/xwiki-platform-index-ui/src/main/resources/XWiki/DocumentsMacro.xml

```diff
@@ -177,6 +177,7 @@ Generates:
 })
 #set($options = { 
   'translationPrefix' : 'platform.index.',
+  'selectedColumn' : 'doc.location',
```

Comment:
The list of columns is configurable so I hope there's no problem if ``doc.location`` is not in the list (for instance when you want to display documents from the same location).

Reply:
I just added the ``selectedColumn``for backwards compatibility reasons (maybe users got used to having the page sorted by location initially ...). I tested and read the code, and this does not cause any errors when the ``doc.location`` column is not present; however in that case no column is selected by default, not even the first one.
I guess I will change the code so it adds the ``selectedColumn`` setting here only if ``doc.location`` is in the list of columns to be displayed.